v2/cli/term/job_page.go

```diff
@@ -103,62 +98,87 @@ func (j *jobPage) fillJobInfo(job core.Job) {
 	j.jobInfo.SetText(info)
 }
 
-func (j *jobPage) fillLogs(eventID, jobName string) {
-	j.logsBox.Clear()
-	go j.streamLog(eventID, jobName)
-}
-
-// nolint: lll
-func (j *jobPage) streamLog(eventID, jobName string) {
-	// // Initialize control channels for the streaming.
-	// j.stopStreaming = make(chan struct{})
-	// j.canStream = make(chan struct{})
-
-	// // Save the context on goroutine.
-	// ss := j.stopStreaming
-	// cs := j.canStream
-	// l := ctx.Log
-
-	// // Close our reader when finished streaming, ignore if error.
-	// defer l.Close()
+func (j *jobPage) fillContainerTable(eventID string, job core.Job) {
```

Comment:
Nit: The UI control in question is named `containersTable` (plural "containers"), so I'd slightly prefer that the function is named `fillContainersTable` for consistency.

Reply:
fwiw... remember "nits" mean the reviewer is acknowledging just how picky they are being. Nits don't generally block your PR from being merged.